he.lproj/Localizable.strings

```diff
@@ -161,73 +161,73 @@
 
 /* Broadcasts v2.0 */
 
-"ERROR_ALERT_TITLE" = "Error: %@";
+"ERROR_ALERT_TITLE" = "שגיאה: %@";
 
-"LIBRARY_STATE_UNAUTHENTICATED" = "Not Logged In";
-"LIBRARY_STATE_QUOTA_EXCEEDED" = "Out of Space";
+"LIBRARY_STATE_UNAUTHENTICATED" = "לא מחובר";
+"LIBRARY_STATE_QUOTA_EXCEEDED" = "שטח האחסון מלא";
 
-"SIDEBAR_CATEGORY_DIRECTORY" = "Browse";
-"SIDEBAR_TITLE_DIRECTORY" = "Directory";
+"SIDEBAR_CATEGORY_DIRECTORY" = "עיון";
```

Comment:
What do we use with all the other verbs? It should all either be "עיון", "שדרוג", "ניגון", or "עיין", "שדרג", "נגן", but not both.